core-bundle/src/Filesystem/Dbafs/Dbafs.php

```diff
@@ -185,18 +185,19 @@ public function setExtraMetadata(string $path, array $metadata): void
         $event = new StoreDbafsMetadataEvent(
             $this->table,
             $row,
-            // Remove non-matching columns before dispatching event
-            array_intersect_key($metadata, $columnFilter)
+            $metadata,
```

Comment:
`$metadata` does not look like the `tl_files` column at this stage. It has key for `metadata` and `importantPart` which are converted by the `DbafsMetadataSubscriber` to `meta` and `importantPartX` etc.
The event must have the original metadata or the subscriber won't work at all.

Reply:
Not sure how this could have slipped through :see_no_evil:. I had tons of test cases in my app, but seems like a last minute refactoring that went bad or something.

The unit tests need to be adjusted as well.

Reply:
done